packages/compartment-mapper/test/fixtures-policy/node_modules/myattenuator/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "myattenuator",
+  "version": "1.0.0",
+  "main": "./index.js",
+  "type": "module",
+  "dependencies": {
```

Comment:
Suspect lint-fix would collapse the {}

Reply:
unfortunately, it refuses to go past a `node_modules` folder. I think I had a workaround for that.